packages/static-module-record/src/hidden.js

```diff
@@ -5,12 +5,14 @@ export const HIDDEN_IMPORT = `${HIDDEN_PREFIX}import`;
 export const HIDDEN_IMPORT_SELF = `${HIDDEN_PREFIX}importSelf`;
 export const HIDDEN_IMPORTS = `${HIDDEN_PREFIX}imports`;
 export const HIDDEN_ONCE = `${HIDDEN_PREFIX}once`;
+export const HIDDEN_META = `${HIDDEN_PREFIX}import_meta`;
```

Comment:
One of us requested that we somehow make the apparent width of the replacement string the same as the original, so it doesn’t displace the column number of following text. I’m not sure how ZWJ generally appears in, say, VSCode, but maybe this should be something like`$h_____meta`.

Reply:
ZWJ can display as nothing or as a blocky box and a warning that  something's fishy about the file. I guess it's the number of characters that counts, so ZWJ counts as 1.

I'll adjust the length and add a comment explaining what happened.

Reply:
length including ZWJj should now match